paper-autocomplete-chips.js

```diff
@@ -365,27 +379,34 @@ class PaperAutocompleteChips extends translatable(PolymerElement) {
 	 */
 	_clearItemSelection(event) {
 		const item = event.model.item,
-			selectedIndex = this.selectedItems.indexOf(item);
-
+			tempIndex = this.selectedItems.indexOf(item.description),
+			selectedIndex = tempIndex !== -1 ? tempIndex : this.selectedItems.indexOf(item);
 		// This will remove from the DOM the source element of the processed event ...
 		this.splice('selectedItems', selectedIndex, 1);
 		// ... so we must prevent further propagation of this event, because its source is now invalid.
 		// (This has caused troubles in app-drawer-layout click event handler).
 		event.preventDefault();
 		event.stopPropagation();
 	}
-	/**
-	 * Get text for the chip labels.
-	 * @param {object} item Item.
-	 * @param {string} textProperty Property in the item where the text is located.
-	 * @returns {void}
-	 */
-	_getChipText(item, textProperty) {
-		let localItem = item;
-		if (typeof localItem === 'object') {
-			localItem = this.get(textProperty, localItem) || localItem.text;
+	enrichChips(itemList, textProperty) {
+		const enrichedChips = itemList.base;
```

Comment:
I wouldn't call itemList.base `enrichedChips`. Semantically the result of the `map` is the `enriched` data, so this is incorrect.
Why not use `itemList.base` directly, instead of putting it in a const?